ADR/Analysis/Performance/tests/test_Power.py

```diff
@@ -0,0 +1,476 @@
+from ADR.Analysis.Performance.Power import *
```

Comment:
It's a good practice to import specifically what you will be using, to prevent import conflicts on star imports (you may end to import a function that overrides another one you are using). In this case you can import the Power class.

Reply:
Ok, for this specific case i've imported everything from the power.py file 'cause i needed the dict_to_dataframe function as well, that it was already imported in Power.py. Calling just the power class the test won't run.

To make the code clearer and to not call unused functions and files with the * call, i've imported separately the dict_to_dataframe function into the test and it worked just fine. Commiting soon.